CMakeLists.txt

```diff
@@ -161,21 +164,6 @@ install(DIRECTORY "${CMAKE_SOURCE_DIR}/include/"
 
 add_subdirectory(src)
 
-if (CFG_PYTHON_BINDINGS)
```

Comment:
moved down so if comes after `enabled_testing()` to register python thests in `CTest`